source/_integrations/frontend.markdown

```diff
@@ -116,6 +116,42 @@ For more details about splitting up the configuration into multiple files, see [
 
 Check our [community forums](https://community.home-assistant.io/c/projects/themes) to find themes to use.
 
+## Theme variables
+
+<div class='note warning'>
+This list is incomplete and should only serve as a starting point. We try to extend this list and keep it up-to-date, but the definitive list of supported variables is the frontend code itself.
```

Comment:
Should there be a link to the fronted repo here?

Reply:
I had thought about that as well (even considered adding links to the most important source files that define the default variable values) but initially landed at: The user group that goes through those user docs (as those are not the dev docs) probably does not want to look at the code, but now that I think about it again, I'd say: Adding that link does not hurt either 😄.

Does adding links to `ha-style.ts` also help, or would that be too much then?